src/libmongoc/src/mongoc/mongoc-cmd.c

```diff
@@ -796,8 +796,8 @@ _txn_in_progress (mongoc_cmd_parts_t *parts)
    }
 
    return (_mongoc_client_session_txn_in_progress (cs)
-	   /* commitTransaction and abortTransaction count as in progress, too. */
-	   || parts->assembled.is_txn_finish);
+      /* commitTransaction and abortTransaction count as in progress, too. */
```

Comment:
Unrelated whitespace change, but the tabs really stood out.

Reply:
Ugh that's from my text editor, I'm not sure why clang-format didn't fix it for me.  Sorry!